sites/skeleton.dev/src/routes/(inner)/docs/contributing/requirements/+page.svelte

```diff
@@ -48,109 +42,112 @@
-		<p>The wildcards (*) should be replaced with short and semantic descriptions that are lowercase and separated by dashes.</p>
-		<pre class="pre">feat/my-new-component-name</pre>
 	</section>
 
-	<!-- Dependencies -->
+	<!-- Monorepo -->
 	<section class="space-y-4">
-		<h2 class="h2">Third Party Dependencies</h2>
+		<h2 class="h2">Monorepo</h2>
 		<p>
-			Avoid adding additional dependencies without prior consent from a core team member. Pull Requests that do not follow this
-			recommendation will be rejected outright.
+			Skeleton uses a monorepo to group related projects into a single repository. This allows us to easily maintain multiple projects and
+			packages in parallel. This includes the Skeleton core library, this documentation site, and more.
 		</p>
+		<!-- prettier-ignore -->
+		<p>
+			In order to contribute to Skeleton, you will be required to install and use <a class="anchor" href="https://pnpm.io/" target="_blank" rel="noreferrer">pnpm</a>, as it fully supports monorepo project structures. Don't worry, it's very similar to <code class="code">npm</code> and will handle most of the heavy lifting.
+		</p>
+		<p>We've provide a quick reference below for where each project is located.</p>
+		<Table source={tableProjectStructure} />
+		<h3 class="h3">Getting Started</h3>
+		<p>If you're not familiar with using a monorepo, please follow the steps below:</p>
+		<!-- prettier-ignore -->
+		<ol class="list-decimal list-inside space-y-2">
+			<li>Install <a class="anchor" href="https://pnpm.io/" target="_blank" rel="noreferrer">pnpm</a> globally using <code class="code">npm install -g pnpm</code>. Confirm via <code class="code">pnpm --version</code> in your terminal.</li>
+			<li>Run <code class="code">git clone https://github.com/skeletonlabs/skeleton.git</code> to clone the monorepo project.</li>
+			<li>Run <code class="code">cd skeleton</code> to move into the cloned monorepo project.</li>
+			<li>Run <code class="code">pnpm i</code> to install depedencies for all projects.</li>
+			<li>Run <code class="code">cd sites/skeleton.dev</code> to move into the Skeleton documentation project.</li>
+			<li>Run <code class="code">code .</code> to open the current project directory in VS Code.</li>
+			<li>Run <code class="code">pnpm dev</code> to start a local dev server. You may see a warning the first time.</li>
+			<li>Open the dev server address provided by the terminal in your browser.</li>
+			<li>Modify the documentation or library components as desired. This fully supports HMR.</li>
+		</ol>
+		<h3 class="h3">Adding New Projects</h3>
+		<p>
+			If you wish to add a new project, run <code class="code">pnpm csa yoursitename</code> from the root of the monorepo. This will
+			generate a brand new Skeleton project via
+			<a class="anchor" href="https://github.com/skeletonlabs/create-skeleton-app" target="_blank" rel="noreferrer">create-skeleton-app</a>
+			in the <code class="code">/sites</code> directory.
+		</p>
```

Comment:
CSA needs to be cloned first with `pnpm getall` from the root of the monorepo.
```suggestion
			If you wish to add a new project, first run <code class="code">pnpm getall</code> from the root of the monorepo. This will clone <a class="anchor" href="https://github.com/skeletonlabs/create-skeleton-app" target="_blank" rel="noreferrer">create-skeleton-app</a> into your project under <code class="code">packages/create-skeleton-app</code>. Next run <code class="code">pnpm csa yoursitename</code>. This will generate a brand new Skeleton project in the <code class="code">/sites</code> directory.
		</p>
```